web/src/components/ChatConversation.tsx

```diff
@@ -49,32 +50,42 @@ const TYPING_INDICATOR_TIMEOUT = 60000
 const ChatConversation = ({ messages, hasError, className }: ChatConversationProps): ReactElement => {
   const { t } = useTranslation('chat')
   const [messagesCount, setMessagesCount] = useState(0)
-  const [typingIndicatorVisible, setTypingIndicatorVisible] = useState(false)
   const messagesEndRef = useRef<HTMLDivElement>(null)
   const isLastMessageFromUser = messages[messages.length - 1]?.userIsAuthor
   const hasOnlyReceivedInfoMessage = messages.filter(message => !message.userIsAuthor).length === 1
   const waitingForAnswer = isLastMessageFromUser || hasOnlyReceivedInfoMessage
+  const [typingIndicatorVisible, setTypingIndicatorVisible] = useState(false)
+
+  const scrollToBottom = async (beforeScroll?: () => void | Promise<void>) => {
+    if (beforeScroll) {
+      await beforeScroll()
+    }
```

Comment:
:wrench: This is never actually used, is it?
```suggestion
  const scrollToBottom = () => {
```